src/command-instance.js

```diff
@@ -5,19 +5,18 @@ const SSH = require("./ssh");
 
 module.exports = class CommandInstance {
     constructor() {
+        this.ssh = new SSH();
+
         this.fileName = this.normalizePath(vscode.window.activeTextEditor.document.fileName);
+        this.fileName = this.ssh.remapLocalPath(this.fileName)
 
         this.methodName = this.findMethodName();
 
         this.executablePath = vscode.workspace.getConfiguration('better-phpunit').get('phpunitBinary')
             || this.findExecutablePath();
+        this.executablePath = this.ssh.remapLocalPath(this.executablePath)
 
         this.shellCommand = `${this.executablePath} ${this.fileName} ${this.filterString()}${this.commandSuffix()}`;
```

Comment:
same with shellCommand

Reply:
First assignment is generating the command, second assignment is wrapping the command **when SSH testing is enabled**.